src/main/io/gps.c

```diff
@@ -877,6 +878,13 @@ void gpsUpdate(timeUs_t currentTimeUs)
         updateGPSRescueState();
     }
 #endif
+
+    static bool hasFix = false;
+    if (STATE(GPS_FIX) && !hasFix) {
```

Comment:
`hasFix` is always false here. Should this be initialized earlier (outside the update)?

Reply:
It's static so it's not. If you see the line below the if test it will be updated with STATE(GPS_FIX). This code just plays the beeps if we get a fix when we didn't have a fix.

Reply:
You are correct that the static variable only is set once.